README.md

```diff
@@ -1 +1,182 @@
-# Math-magician
\ No newline at end of file
+# Math-magician
+<a name="readme-top"></a>
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍